pkg/operator/e2e/context.go

```diff
@@ -90,7 +92,7 @@ func newTestContext(t *testing.T) *testContext {
 	t.Cleanup(cancel)
 	t.Cleanup(func() { tctx.cleanupNamespaces() })
 
-	tctx.ownerReferences, err = tctx.createBaseResources()
+	tctx.ownerReferences, err = tctx.createBaseResources(context.TODO())
```

Comment:
Never really understood when to use this vs `context.Background()`.. any ground rules you go by?

Reply:
There's no technical difference besides Background being permanent while TODO indicating this should be connected to a higher level context eventually.